### DISCUSSION TITLE
Markdown: links target _blank

### DISCUSSION THREAD

**Discussion by shapkarin** [community/community#102510] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

It would be nice to have such feature for any project `*.md` files as well as comments.
Syntax example from @dpolivaev at the [stackoverflow discussion](https://stackoverflow.com/a/63546509):
```md
[go](http://stackoverflow.com){:target="_blank" rel="noopener"}
```

ref: [stackoverflow: open link in new tab with github markdown using target="_blank"](https://stackoverflow.com/questions/41915571/open-link-in-new-tab-with-github-markdown-using-target-blank )

**Comment by ghost** [comment]
This looks great

**Comment by alihamza-developer** [comment]
Response #100001

**Reply by shapkarin** [reply]
@alihamza-developer what do you mean?

**Reply by ghost** [reply]
Its like he supports your opinion

**Comment by MasterInQuestion** [comment]
Supporting [ [https://github.com/orgs/community/discussions/68734#discussioncomment-7347910](https://github.com/orgs/community/discussions/68734#discussioncomment-7347910) ] first (so to address the underline interference issue) appears of more significance.

    Caveat the compatibility with existing content when implementing:
    The design may cause notable unwanted breakage.

    Probably would be better to support:
[[
```html
	<a href="https://example.com/" target="_blank">Website</a>
```
]]
(just additionally reckon the ["target"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) attribute)

**Reply by MasterInQuestion** [reply]
Follow-Up: https://github.com/orgs/community/discussions/102510#discussioncomment-8730420
    @shapkarin, I believe much is already clear in the post.

    For the compatibility issue, consider the worst case:
    https://old.reddit.com/r/redditdev/comments/1b91ymx?sort=old#ktthty2
    (embedding RegEx alike in Markdown)

**Comment by shapkarin** [comment]
@MasterInQuestion please provide working markdown example. 
You may use https://gist.github.com to show it.
I was not able to implement the same with Markdown – `target="_blank"  (open link at another tab)`

Sure, of course it's possible to use just HTML include
```markdown
<a href="https://example.com/" target="_blank">Website</a>
```
But this thread is about Markdown.

**Reply by MasterInQuestion** [reply]
Probably you misunderstood the topic..?
    Which is about creator configured link underlining. (a practical proposal)

    Markdown may include much HTML.

**Reply by shapkarin** [reply]
@MasterInQuestion sure, but how to do the same by Markdown?

**Reply by MasterInQuestion** [reply]
Markdown-style equivalent is not always necessary.

**Reply by shapkarin** [reply]
@MasterInQuestion html is as one of solutions and at least it's not in the "GitHub Flavored Markdown Spec"  https://github.github.com/gfm/#links that's discussions more about GitHub Flavored Markdown. Maybe it's open source like https://github.com/commonmark

**Reply by MasterInQuestion** [reply]
https://github.github.com/gfm/#raw-html
    https://github.github.com/gfm/#html-blocks

**Reply by shapkarin** [reply]
@MasterInQuestion I know but it's about MD-style itself. Of course I can use HTML. 
BTW, even HTML and `target="_blank"` does not works well. Markdown just ignore that attribute.
Please check my https://github.com/shapkarin/shapkarin

This:
```md
- GitHub community – <a href="https://github.com/orgs/community/discussions?discussions_q=author%3Ashapkarin+" target="_blank">GitHub/community/discussions :octocat:</a>
```
Becomes:
```html
<a href="https://github.com/orgs/community/discussions?discussions_q=author%3Ashapkarin+">GitHub/community/discussions <img class="emoji" title=":octocat:" alt=":octocat:" src="https://github.githubassets.com/images/icons/emoji/octocat.png" height="20" width="20" align="absmiddle"></a>
```

UPD: https://github.com/orgs/community/discussions/102510#discussioncomment-8715363

**Comment by shapkarin** [comment]
Markdown: links in a new tab are ignored even when HTML blocks are used #102510

**Comment by egel** [comment]
I wish there would be also support for links with link-references:

```markdown
[go][my-go-link]{:target="_blank" rel="noopener"}

[my-go-link]: http://stackoverflow.com
```
